src/test/unit/ntypes/ArrayTest.cpp

```diff
@@ -31,16 +31,18 @@
 #include <nupic/ntypes/Array.hpp>
 #include <nupic/os/OS.hpp>
 
-
 #include <map>
 #include <memory>
 #include <limits.h>
 
 #include <gtest/gtest.h>
 
 namespace testing {
-    
+
+static bool verbose = false;
+#define VERBOSE if(verbose) std::cerr << "[          ]"
```

Comment:
OT: can we make some meta-include, so this is included in all tests? Maybe even useful for non-test classes, add to Log.hpp? (consider in a separate Issue)

Reply:
> static bool verbose = false;
#define VERBOSE if(verbose) std::cerr << "[          ]"
can we make some meta-include, so this is included in all tests? 

Well, the `[             ]` part is intended to match up with the gtest output so that your text does not mess up the `[     OK]` column.  I don't think this applies much outside of gtest.  You would not want the `static bool verbose` to be global.  

So, lets not make this common.  We can add these two lines if we want to use this VERBOSE macro in a test.